packages/@aws-cdk/cdk/lib/app.ts

```diff
@@ -184,61 +147,13 @@ export class App extends Root {
     return stack;
   }
 
-  private runCommand() {
-    switch (this.request!.type) {
-      case 'list':
-        return {
-          stacks: this.listStacks()
-        } as cxapi.ListStacksResponse;
-
-      case 'synth':
-        return {
-          stacks: this.synthesizeStacks((this.request as cxapi.SynthesizeRequest).stacks),
-          runtime: this.collectRuntimeInformation()
-        } as cxapi.SynthesizeResponse;
-
-      default:
-        throw new Error(`Invalid command: ${this.request!.type}`);
-    }
-  }
-
-  private get usage() {
-    const progname = this.progname ? this.progname + ' ' : '';
-
-    return `${CX_VERSION}
-
-Usage:
-  ${progname}REQUEST
-
-REQUEST is a JSON-encoded request object.
-`;
-  }
-
   private loadContext() {
-    const context = (this.request && this.request.context) || {};
+    const contextJson = process.env[cxapi.CONTEXT_ENV];
+    const context = !contextJson ? { } : JSON.parse(contextJson);
```

Comment:
About as awkward, but doesn't use a ternary: `JSON.parse(contextJson || '{}')`?

Reply:
Prepare the ternary